src/theVoice/TheVoice.style.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components'
+
+import { color, responsiveBreakpoints, space } from '../_utils/branding'
+import { normalizeHorizontally } from '../layout/layoutNormalizer'
+import { TheVoice } from './TheVoice'
+
+const StyledTheVoice = styled(TheVoice)`
+  /* @TODO: Align scale and spec
+  /* 32px it's not part of current scale. 4 it could be calculated by the base unit space.s  */
+  --space-bottom: 32px;
+  /* 40px is an exception on font scale  */
+  --font-size-desktop: 40px;
+
+  margin: 0;
+  padding: ${space.xl} 0 var(--space-bottom);
+  color: ${props => (props.isInverted ? color.white : '')};
+  white-space: pre-line;
+
+  ${normalizeHorizontally};
+
+  @media (${responsiveBreakpoints.isMediaLarge}) {
+    text-align: center;
+    font-size: var(--font-size-desktop);
+  }
+`
+
+export default StyledTheVoice
```

Comment:
no need to default export, better keep it as a named export

Reply:
You're right, done! :)